lib/account/account-manager.js

```diff
@@ -36,6 +75,25 @@
  */
 
 /**
+ * Lifetime for how long a file should last. This can
+ * contain as many properties as needed to adjust the
+ * time.
+ *
+ * @typedef {Object} accountManager~lifeTime
+ * @property {string} Hours, Minutes, Seconds
+ *
+ */
+
+/**
+ * Information the Account Holder sends to complete the
+ * login process.
+ *
+ * @typedef {Object} accountManager~loginCompleteInfo
+ * @property {string} password
```

Comment:
When I read "password" I expect that it is a plain text password.
When I read "passwordPbkdf2Hash" and "secureHash" I do not expect a plain text password.

I think that naming it appropriately is more understandable.